sites/geohub/src/routes/(map)/dashboards/ceei/+page.svelte

```diff
@@ -0,0 +1,253 @@
+		const ceeiTopojsonGzipUrl = 'https://undpgeohub.blob.core.windows.net/ceei/ceei.topojson.gz';
+		const ceeiTopojsonGzipRes = fetch(ceeiTopojsonGzipUrl)
+			.then((res) => res.arrayBuffer())
+			.then((buff) => {
+				const ceeiTopojson = JSON.parse(
+					pako.ungzip(buff, {
+						to: 'string'
+					})
+				);
+
+				return {
+					ceeiJson: topojson.feature(ceeiTopojson, ceeiTopojson.objects.admin2),
+					ceeiMapBbox: topojson.bbox(ceeiTopojson)
+				};
+			});
+
+		const ceeiDataUrl = 'https://undpgeohub.blob.core.windows.net/ceei/original/raw_data_ceei.xlsx';
+		const ceeiDataXlsxRes = fetch(ceeiDataUrl).then((res) => res.arrayBuffer());
+
+		const countriesUrl = 'https://geohub.data.undp.org/api/countries';
+		const countriesRes = fetch(countriesUrl).then((res) => res.json());
+
+		const ceeiDataXlsx = await ceeiDataXlsxRes;
+		const ceeiWorkbook = read(ceeiDataXlsx);
+		let ceeiData = utils.sheet_to_json(ceeiWorkbook.Sheets[ceeiWorkbook.SheetNames[0]]);
+
+		const countries = await countriesRes;
+		countriesList = countries
+			.filter((c) => c.continent_name === 'Africa')
+			.map((c) => {
+				return {
+					label: c.country_name,
+					value: c.country_name
+				};
+			});
+		countriesList.unshift({
+			label: 'All',
+			value: 'All'
+		});
+		ceeiData = ceeiData.map((d) => {
+			const newRow = {};
+
+			Object.keys(d).forEach((k) => {
+				const newHeader = headerMapping[k];
+				if (newHeader) {
+					newRow[newHeader] = d[k];
+				}
+			});
+
+			const { country_name } = countries.find((c) => c.iso_3 === d.iso3_country_code);
+			newRow['Country'] = country_name;
+			newRow['CEEI'] = Math.random(); // TEMP VALUE UNTIL CEEI IS COMPUTED
+			return newRow;
+		});
+
+		const defaultColorMap = 'rdylbu';
+
+		const { ceeiJson, ceeiMapBbox } = await ceeiTopojsonGzipRes;
+		return {
+			name: 'Base Layer',
+			isVisible: true,
+			sourceId: 'CEEI' + '-source',
+			bounds: ceeiMapBbox,
+			source: {
+				type: 'geojson',
+				data: ceeiJson,
+				promoteId: 'adminid'
+			},
+			layerId: 'CEEI' + '-layer',
+			layer: {
+				id: 'CEEI' + '-layer',
+				type: 'fill',
+				source: 'CEEI' + '-source',
+				layout: {
+					visibility: 'visible'
+				},
+				paint: {
+					'fill-opacity': 0.7
+				}
+			},
+			isMapLoaded: false,
+			isDataLoaded: false,
+			data: ceeiData as object[],
+			colorMap: defaultColorMap
+		};
+	};
+
+	const handleCountryFilter = (e) => {
+		const selectedItem = e.detail.item.value;
+		if (selectedItem === 'All') {
+			$layerStore.forEach((l) => {
+				map.setFilter(l.layerId, undefined);
+			});
+		} else {
+			$layerStore.forEach((l) => {
+				map.setFilter(l.layerId, ['==', 'Country', selectedItem]);
+			});
+		}
+	};
+
+	onMount(async () => {
+		let protocol = new pmtiles.Protocol();
+		addProtocol('pmtiles', protocol.tile);
+
+		map = new Map({
+			container: mapContainer,
+			style: styles[0].uri,
+			center: [0, 0],
+			zoom: 2.5,
+			hash: true,
+			attributionControl: false
+		});
```

Comment:
@lrt-sibat can you set maxBounds to maplibre Map object to prevent users to move to outside of Africa?

https://maplibre.org/maplibre-gl-js/docs/API/type-aliases/MapOptions/#maxbounds